google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -572,7 +572,7 @@ public void checkAccess(Path path, AccessMode... modes) throws IOException {
     // null size indicate a file that we haven't closed yet, so GCS treats it as not there yet.
     if (null == blobInfo || blobInfo.getSize() == null) {
       throw new NoSuchFileException(
-          cloudPath.getBlobId().getBucket() + "/" + cloudPath.getBlobId().getName());
+          "gs://" + cloudPath.getBlobId().getBucket() + "/" + cloudPath.getBlobId().getName());
```

Comment:
Why not use `String.format` like the other referenced error message?

https://github.com/GoogleCloudPlatform/google-cloud-java/blob/f72746c6183378664a9326ee406d679926a7def4/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageReadChannel.java#L196